web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss

```diff
@@ -22,12 +22,15 @@
     }
 
     &.usa-current {
-      @include vote-link-bg($color: $base-dark);
+      // @include vote-link-bg($color: $base-dark);
+      @include vote-link-bg; // temporarily using default color
 
+      // section indicator (left of current section heading)
```

Comment:
i dont think this comment is needed... i would change to something like. "Turning off style untill USWDS fix is made"